packages/liborc/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_DESCRIPTION="Library of Optimized Inner Loops Runtime Compiler"
```

Comment:
Where is the homepage?

Reply:
Sorry i will add for both liborc and libspice-protocol.
Also i miss understood pkg_breaks and pkg_replaces